backend/papersfeed/models/subscription/subscription.py

```diff
@@ -0,0 +1,49 @@
+"""subscription.py"""
+from django.db import models
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from papersfeed.models.users.user import User
+
+class Subscription(models.Model):
```

Comment:
Please inherit BaseModel. And use creation_date of it instead of your own. And when you make migration files multiple times in one PR, delete all newly generated(by you) migration files, then make one migration file.

Reply:
change reflected.